packages/client/hmi-client/src/composables/useCharts.ts

```diff
@@ -130,10 +130,10 @@ export function useCharts(
 		setting: ChartSettingEnsembleVariable,
 		modelConfigId = '',
 		multiVariable = false,
-		showBaseLines = false
+		showBaseLines = false,
+		color: string | null = null
```

Comment:
Same here, `color` argument is not needed since you can access from `setting.primaryColor`